runner/ollamarunner/cache.go

```diff
@@ -267,7 +277,25 @@ func (c *InputCache) ShiftCacheSlot(slot *InputCacheSlot, numKeep int32) error {
 	if c.cache != nil {
 		err := c.cache.Remove(slot.Id, numKeep, numKeep+discard)
 		if err != nil {
-			return fmt.Errorf("unable to remove old kv cache entries (id: %v, keep: %v discard: %v): %w", slot.Id, numKeep, discard, err)
+			slog.Debug("kv cache removal failed, clearing cache and returning inputs for reprocessing",
```

Comment:
I can't check for a specific error here yet, since we don't have a model which will fail. We can bring that logic in once we have a model implemented which does not support kv cache shifting.

Reply:
Good suggestion, I tested with this and things seem good.